contracts/main/price-feeders/DelayFathomOraclePriceFeed.sol

```diff
@@ -8,29 +8,27 @@ import "../interfaces/IFathomOracle.sol";
 import "../interfaces/IAccessControlConfig.sol";
 
 contract DelayFathomOraclePriceFeed is PausableUpgradeable, IFathomOraclePriceFeed {
-    struct Feed {
-        uint256 price;
-        uint256 lastUpdateTS;
-    }
-
-    Feed public currentPrice;
+    uint256 public latestPrice;
+    uint256 public delayedPrice;
+    uint256 public lastUpdateTS;
     uint256 public timeDelay;
+    uint256 public priceLife;
 
     IFathomOracle public fathomOracle;
     IAccessControlConfig public accessControlConfig;
     address public token0;
     address public token1;
-    uint256 public priceLife; // [seconds] how old the price is considered stale, default 1 day
 
     function initialize(address _fathomOracle, address _token0, address _token1, address _accessControlConfig) external initializer {
         PausableUpgradeable.__Pausable_init();
 
+        require(_accessControlConfig != address(0), "FathomOraclePriceFeed: ZERO_ADDRESS");
         accessControlConfig = IAccessControlConfig(_accessControlConfig);
 
         fathomOracle = IFathomOracle(_fathomOracle);
-        require(_token0 != _token1, "FathomOraclePriceFeed/wrong-token0-token1");
-        token0 = _token0;
-        token1 = _token1;
+        require(_token0 != _token1, "FathomOraclePriceFeed/same-token0-token1");
+        require(_token0 != address(0) && _token1 != address(0), "FathomOraclePriceFeed: ZERO_ADDRESS");
+        (token0, token1) = _token0 < _token1 ? (_token0, _token1) : (_token1, _token0);
```

Comment:
2.2.2 There is no validation in DelayFathomOraclePriceFeed - fix, added tokens sorting